src/marshmallow/fields.py

```diff
@@ -1314,34 +1314,29 @@ def _deserialize(self, value, attr, data, **kwargs):
         return ret
 
 
-class Time(Field):
-    """ISO8601-formatted time string.
+class Time(DateTime):
```

Comment:
thought (non-blocking): It feels a bit odd to make `Time` a subclass of `DateTime`.

But I suppose this is consistent with `Date` being a subclass of `DateTime`.